### DISCUSSION TITLE
AdGuard Home - No update notification / option

### DISCUSSION THREAD

**Discussion by Turbine1313** [AdguardTeam/AdGuardHome#4518] [Q&A]
Hey Guys

I have AdGuard Home installed on Orange pi 3 LTS, via Portainer

On the bottom right I have current version :  v0.107.5 , I saw there is new version of : [v0.107.6](https://github.com/AdguardTeam/AdGuardHome/releases/tag/v0.107.6)

Why I don't have an update notification or an update option ?

And how can i update the AdGuard manually ?

thanks :)

**Comment by ainar-g** [comment]
Hello.  We have a manual update guide there: <https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#manual-update>.  I'm not sure why the notification about the update didn't appear for you, but perhaps the AdGuard servers weren't reachable?
